src/main/java/com/flowingcode/vaadin/addons/chatassistant/ChatAssistant.java

```diff
@@ -58,6 +62,29 @@ public void sendMessage(String message) {
   public void toggle() {
     getElement().executeJs("setTimeout(() => {this.toggle();})");
   }
+  
+  /**
```

Comment:
This is a fragment—a noun phrase or verb phrase, not a complete sentence. It does not begin with `A {@code Foo} is a...`, or `This method returns...`

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment